src/Cache/Cache.php

```diff
@@ -94,13 +94,13 @@ public function getOrSet(string $key, $ttl, callable $function, array $arguments
      *
      * @param string $className
      * @param string $functionName
-     * @param \string[] ...$uniqueValues
+     * @param \null[]|\string[] ...$uniqueValues
      * @return string
      */
-    public function keyHelper(string $className, string $functionName, string ...$uniqueValues): string
+    public function keyHelper(string $className, string $functionName, ?string ...$uniqueValues): string
     {
         // Please help prevent cache namespace collisions by driving carefully
-        $uniqueValues = str_replace('.', '_', $uniqueValues);
+        $uniqueValues = str_replace('.', '_', array_filter($uniqueValues));
```

Comment:
Does this cause potential for collisions if there are multiple arguments that could be null?

Setting $uniqueValues to`['foo', 1, null]` or `['foo', null , 1]` would both result in the same key. Those empty values should be set to some placeholder value rather than stripped out entirely to avoid that collision.